vars/checkPatch.groovy

```diff
@@ -41,11 +61,24 @@ def call(Map config) {
                  context: "pre-build" + "/" + env.STAGE_NAME,
                  status: "PENDING"
 
-    script = 'GH_USER="' + config['user'] + '"' + \
-             ' GH_PASS="' + config['password'] + '"' + \
-             ' CHECKPATCH_IGNORED_FILES="' + ignored_files + '"' + \
-             ' jenkins/code_review/jenkins_github_checkwarn.sh'
-    rc = sh(script: script, returnStatus: true)
+    rc = 1
+    if (config['review_creds']) {
+      withCredentials([[$class: 'UsernamePasswordMultiBinding', 
+                      credentialsId: config['review_creds'],
+                      usernameVariable: 'GH_USER',
+                      passwordVariable: 'GH_PASS']]) {
+        script = 'CHECKPATCH_IGNORED_FILES="' + ignored_files + '"' + \
+                 ' jenkins/code_review/jenkins_github_checkwarn.sh'
+        rc = sh(script: script, returnStatus: true)
+      }
+    } else {
+      // Older method to be deprecated
```

Comment:
Or can be left for cases where somebody doesn't want to use a Jenkins credential.